classes/mutex/SpinlockMutex.php

```diff
@@ -79,11 +80,13 @@ protected function unlock()
         $elapsed = microtime(true) - $this->acquired;
         if ($elapsed >= $this->timeout) {
             $message = sprintf(
-                "The code executed for %d seconds. But the timeout is %d seconds.",
+                "The code executed for %.2f seconds. But the timeout is %d " .
```

Comment:
Thanks. Great idea upgrading to add fractions of seconds too.

Can you switch it to use `%.2F` so that the formatting is not locale aware.

Reply:
Sure, did not know about this. Thanks!